functions/task2.js

```diff
@@ -0,0 +1,10 @@
+const ageOfAdulthood = 18;
+
+function isPersonAdult(age) {
```

Comment:
it's not so bad to use this name `isPersonAdult` but it would be better to write function name as an action like `getIsAdult`

Reply:
It's common practice to use function names like `isSometing`, for example Lodash (isNumber, isNan etc.), validation libs (isEmail, isInt etc.)